simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/parser/rewrite/OutputFormat.java

```diff
@@ -18,18 +18,27 @@
 import java.io.BufferedWriter;
 import java.io.IOException;
 
+import com.github.benmanes.caffeine.cache.simulator.policy.AccessEvent;
+
 /**
  * The trace output format.
  *
  * @author ben.manes@gmail.com (Ben Manes)
  */
 public enum OutputFormat {
   CLIMB {
-    @Override public void write(BufferedWriter writer, long event) throws IOException {
-      writer.write(Long.toString(event));
+    @Override public void write(BufferedWriter writer, AccessEvent event) throws IOException {
+      writer.write(Long.toString(event.key()));
+      writer.newLine();
+    }
+  },
+  ADAPT_SIZE {
+    long time = 0;
+    @Override public void write(BufferedWriter writer, AccessEvent event) throws IOException {
+      writer.write(Long.toString(time++) + " " + Long.toString(event.key()) + " " + Integer.toString(event.weight()));
```

Comment:
another nit.. :)

To avoid the extra string concat just to throw it away, append as multiple statements.